### PR TITLE
Migrate prokka testing to nf-test

### LINKED ISSUE #4458: Port prokka to nf-test
### Is your feature request related to a problem? Please describe

Updating prokka testing from pytest to the nf-test framework

### Describe the solution you'd like

Create a new test folder in prokka module and adapt the pytest script for nf-test

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Puumanamana**
## PR checklist

Closes #4458 

Migrated prokka tests from pytests to nf-test

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/prokka/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// nf-core modules test prokka
```
Comment (LeuThrAsp) [inline modules/nf-core/prokka/tests/main.nf.test:1]:
```suggestion
```

**Review by LeuThrAsp — COMMENTED** [review]
Remove pytest files

**Review by LeuThrAsp — APPROVED** [review]
Looks good :)

**Comment by SPPearce** [comment]
Looks like @louperelo did the same work in #4602 as this wasn't merged. Looks pretty similar to me, do we close this one?

**Comment by Puumanamana** [comment]
> Looks like @louperelo did the same work in #4602 as this wasn't merged. Looks pretty similar to me, do we close this one?

That's fine for me

**Pull request closed without merging**